cmd/client/main.go

```diff
@@ -91,6 +61,50 @@ func main() {
 
 /* ----- Helper functions ----- */
 
+func parseCommandLine() (string, string, int64, int64, []string, int64) {
+	// Optional flags
+	argsPtr := flag.String("args", "NONE", "Command line args for file")
+	rangePtr := flag.String("range", "NONE", "Range for job")
```

Comment:
Provide an example for this one especially

Reply:
for the range or the args?

Reply:
I think having an example for both would be ideal